docs/resources/mws_permission_assignment.md

```diff
@@ -0,0 +1,65 @@
+---
+subcategory: "Unity Catalog"
+---
+# databricks_mws_permission_assignment Resource
+
+These resources are invoked in the account context. Provider must have `account_id` attribute configured.
+
+## Example Usage
+
+In account context, adding account-level group to a workspace:
+
```

Comment:
I see that we only have docs for updating role assignment in the account context. I think thats fine and sufficient but it will not work for workspace admin personas who are not account admins.

Reply:
yes, we'll add that "workspace level account scim user listing" in one of the following releases.